docs/source/faq/index.rst

```diff
@@ -80,24 +97,225 @@ of these popular operating systems from the past few years.
 We also provide :ref:`custom install instructions <alternative-builds>` to use
 FiftyOne on old-but-popular setups like Ubuntu 16.04 and Debian 9.
 
-Can you share a dataset with someone else?
-------------------------------------------
+.. _faq-share-dataset-export:
```

Comment:
There were three separate FAQ questions about sharing datasets. This one, and the ones about collaborating programmatically and non-programatically. I think it's more intuitive from an FAQ standpoint for people to search for "sharing" and then discover all the relevant options for sharing under one question heading.